pkg/rbac/rbac.go

```diff
@@ -142,30 +140,6 @@ func NewEnforcer(ctx context.Context, kubeClient *kubernetes.Kubernetes, l *zap.
 	return enforcer, refreshEnforcerInBackground(ctx, kubeClient, enforcer, l)
 }
 
-// To understand why we need this custom globMatch, consider the below policy:
-// ```
-// p, adminrole:role, database-clusters, read, */*
-// ```
-// Given a request `adminrole:role read database-clusters *`, the default globMatch returns false.
```

Comment:
We never actually get '*' from the request side. I was testing all this while using the `can` command, where a `*` is a valid request parameter, so I just handled the case there internally to make it resemble a real enforce request (like at the API level)